Statistics/Distributions/IDistributionFactory.cs

```diff
@@ -228,7 +229,8 @@ public static IDistribution FactoryTruncatedDistribution(IDistribution distribut
             switch (distribution.Type) 
             {
                 case IDistributionEnum.Normal:
-                    return FactoryTruncatedNormal(distribution.Mean, distribution.StandardDeviation, min, max, distribution.SampleSize);
+                    Statistics.Distributions.Normal ndist = (Statistics.Distributions.Normal)distribution;
```

Comment:
can we please use normalDistribution instead of ndist

Reply:
yes. feel free to update